ReferenceDataApi.Tests/V1/E2ETests/Steps/GetReferenceDataSteps.cs

```diff
@@ -0,0 +1,80 @@
+using FluentAssertions;
+using ReferenceDataApi.V1.Boundary.Response;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace ReferenceDataApi.Tests.V1.E2ETests.Steps
+{
+    public class GetReferenceDataSteps : BaseSteps
+    {
+        public GetReferenceDataSteps(HttpClient httpClient) : base(httpClient)
+        { }
+
+        public async Task WhenReferenceDataIsRequested(string category, bool? includeInactive)
+        {
+            await WhenReferenceDataIsRequested(category, null, includeInactive).ConfigureAwait(false);
+        }
+        public async Task WhenReferenceDataIsRequested(string category, string subCategory, bool? includeInactive)
```

Comment:
Maybe we can call this method CallApi(...) so we don't have 2 methods called the same thing (unless I've misunderstood something here)

Reply:
There is an overload to this method because BDDfy does not allow the use of methods with defaulted input parameters - which is what I would normally do. So because the same functionality is required with slightly different inputs in more than one place, we need to create overloads to replicate what the defaulted parameters would normally provide.

Reply:
Ah okay makes sense